convex/actions/algolia.ts

```diff
@@ -0,0 +1,30 @@
+// Index discord into algolia
+"use node";
```

Comment:
FYI these don't need to be in the actions/ directory anymore. i just haven't moved them since there's two files named discord.js in addition to the library called "discord.js" - i need better names